drift_dev/lib/src/writer/manager/table_manager_writer.dart

```diff
@@ -88,11 +88,14 @@ class _TableManagerWriter {
           .where((fieldName) => fieldName == relation.fieldName)
           .length;
       if (fieldNameCount != 1) {
-        print(
-            "Duplicate orderings/filters detected for field \"${relation.fieldName}\" on table \"${table.entityInfoName}\"."
-            " Filter and orderings for this field wont be generated."
-            " Use the @ReferenceName() annotation to resolve this issue."
-            " See https://drift.simonbinder.eu/docs/manager/#name-clashes for more information");
+        if (table.id.isDefinedInDart) {
```

Comment:
I've disabled the warnings for tables defined in `.drift` files since there currently is no way to fix it.

Reply:
That makes sense